README.md

```diff
@@ -75,3 +75,17 @@ You can run the tests by running:
 ```bash
 $ poetry run pytest
 ``` 
+
+## Deploying the App to a Virtual Machine
+
+To deploy the app using virtual machines - you will need a control node VM and a host VM (or multiple hot VMs).
+
+Install ansible on the control node, copy the files in the ansible folder into the control node VM.
+Add the IP address(es) of your host node(s) to the `inventory.yaml` file
+Open a terminal in the control node and run:
+```bash
+$ ansible-playbook playbook.yml -i inventory.yml
+```
+You will need to have the `TRELLO_API_KEY` and `TRELLO_API_TOKEN` available, and input htem when prompted.
```

Comment:
Similar here 🙂 
```suggestion
You will need to have the `TRELLO_API_KEY` and `TRELLO_API_TOKEN` available, and input them when prompted.
```

Reply:
Less relevant here where our project is primarily a learning opportunity, but if typos are something that occasionally trip you up, it could be interesting to see if there's a simple spell-checker you can run documentation through (I know we had a spell checker built into our CI for some of our learning material repos, but they obviously matter a bit more than your typical repo!)